website/app/components/mdx/Tabs.tsx

```diff
@@ -123,7 +123,8 @@ function extractTabItems(children: React.ReactNode): TabItemElement[] {
       // @ts-ignore access private name which works on production
       const name = child.type.name;
 
-      if (name === 'TabItem') {
+      // in production the element gets minified to 'q4' instead of 'TabItem'
+      if (name === (process.env.NODE_ENV === 'production' ? 'q4' : 'TabItem')) {
```

Comment:
Not sure this is the best way to do this, it may be random that it's `q4` - it could be something else

cc @Ehesp

Reply:
Yeah that's not going to work consistently. 

Can probably just remove the check, it's just there to check TabItem is the direct child but I think if it's being minified we will just have to assume.

Reply:
can remove this change now and sync with main